pkg/ccl/changefeedccl/show_changefeed_jobs_test.go

```diff
@@ -213,22 +213,28 @@ func TestShowChangefeedJobsRedacted(t *testing.T) {
 			sqlDB.QueryRow(t, createStmt).Scan(&jobID)
 			var sinkURI, description string
 			sqlDB.QueryRow(t, "SELECT sink_uri, description from [SHOW CHANGEFEED JOB $1]", jobID).Scan(&sinkURI, &description)
+			sqlDB.Query(t, "CANCEL ALL CHANGEFEED JOBS")
+			testutils.SucceedsWithin(t, func() error {
+				var status string
+				sqlDB.QueryRow(t, "SELECT status FROM [SHOW CHANGEFEED JOB $1]", jobID).Scan(&status)
+				if status != "canceled" {
+					return errors.Newf("expected job status canceled, got %s", status)
+				}
+				return nil
+			}, 1*time.Minute)
 			replacer := strings.NewReplacer(apiSecret, "redacted", certSecret, "redacted")
 			expectedSinkURI := replacer.Replace(tc.uri)
```

Comment:
Not code that you wrote but could you delete the unused `expectedSinkURI` and `expectedDescription` fields in the test cases?